src/views/TodoList.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div id="todo">
+    <h1>Todoリスト</h1>
+    <table id="table">
+      <thead>
+        <tr>
+          <td>
+            <input type="radio" name="check" value="全て" checked="checked"
+            />全て
+          </td>
+          <td><input type="radio" name="check" />作業中</td>
+          <td><input type="radio" name="check" />完了</td>
+        </tr>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="todo in todos" v-bind:key="todo.id">
+          <td>{{ todo.id }}</td>
+          <td>{{ todo.comment }}</td>
+          <td>
+            <button class="btn">
+              {{ todo.status }}
+            </button>
+            <button class="btn">
+              削除
+            </button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <h2>新規タスクの追加</h2>
+    <form id="form" v-on:submit.prevent="addTodo">
+      <input type="text" ref="comment" />
+      <input type="submit" class="addBtn" value="追加" />
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  el: '#todo',
+  data() {
+    return {
+      todos: [],
+    };
+  },
+  methods: {
+    addTodo: function() {
```

Comment:
こういった記法もありますので、覚えておきましょう。
```suggestion
    addTodo() {
```

Reply:
ありがとうございます！